.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish orquestra-sdk-base Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_version:
+        type: string
+        description: Orquestra SDK version to build the image with
+jobs:
+  trigger-build-and-push:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Trigger cicd-actions repository workflow over Github API
+        run: |
+          curl \
+          "https://api.github.com/repos/zapatacomputing/cicd-actions/dispatches" \
+          -H "Authorization: token "$USER_TOKEN \
+          -H 'Accept: application/vnd.github.everest-preview+json' \
+          --data-raw '
+          {
+            "event_type":  "'"$repository"' | '"$ref"'", 
+            "client_payload": 
+                {
+                  "repository": "'"$repository"'", 
+                  "ref": "'"$ref"'", 
+                  "path_to_dockerfile": "'"$path_to_dockerfile"'", 
+                  "docker_context_path": "'"$docker_context_path"'",
+                  "target_docker_repository": "'"$target_docker_repository"'",
+                  "github_sha": "'"$github_sha"'",
+                  "build_number": "",
+                  "additional_docker_build_properties": {
+                    "additional_image_tags": "'"$additional_image_tags"'",
+                    "image_tag_flavor": "'"$image_tag_flavor"'",
+                    "image_labels": "'"$image_labels"'",
+                    "docker_build_args": "'"$docker_build_args"'",
+                    "target_platforms": "'"$target_platforms"'"
+                  }
+                }
+            }
+          '
+        env:
+          USER_TOKEN: ${{ secrets.PAGES_TOKEN }}
+          repository: ${{github.repository}}
+          ref: ${{ github.ref }}
+          path_to_dockerfile: './Dockerfile'
```

Comment:
There's no `Dockerfile` yet, is it OK to test without it?

Reply:
I will test it against my own branch. This PR is just for overcoming a wart in GitHub's UI where an action doesn't show up in the UI until there's a workflow for it in the main branch (even though you execute it on another branch).